src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -205,18 +205,23 @@
 (def user-message-content (quo.theme/with-theme user-message-content-internal))
 
 (defn on-long-press
-  [message-data context keyboard-shown?]
+  [{:keys [deleted? deleted-for-me?] :as message-data} context keyboard-shown?]
   (rf/dispatch [:dismiss-keyboard])
   (rf/dispatch [:show-bottom-sheet
-                {:content       (drawers/reactions-and-actions message-data context)
-                 :selected-item (fn []
-                                  [rn/view {:pointer-events :none}
-                                   [user-message-content
-                                    {:message-data    message-data
-                                     :context         context
-                                     :keyboard-shown? keyboard-shown?
-                                     :show-reactions? true
-                                     :show-user-info? true}]])}]))
+                (cond-> {:content (drawers/reactions-and-actions message-data context)}
```

Comment:
seems like cond is redundant, `if` can be used for :selected-item value

Reply:
fixed, thanks